src/client/utils/throttle.js

```diff
@@ -0,0 +1,31 @@
+import nativeMethods from '../sandbox/native-methods';
+
+export default function throttle (func, ms) {
+    var isSuspend = false;
```

Comment:
After an additional investigation I found that cookie synchronization every <value> ms is a bad choise.
We can get out of cookie sync for the following case:
```javascript
document.cookie = 'key1=value1';
document.cookie = 'key2=value2';
xhrRequest with cookie (or another request with cookie).
```